src/blocks/product-card.css

```diff
@@ -0,0 +1,104 @@
+html {
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+}
+
+.product-card__image-box {
+  margin: 0 auto;
+  margin-bottom: 36px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__name {
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  font-weight: 500;
+}
+
+.product-card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.product-card__view {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: flex-end;
+  align-content: flex-end;
+  line-height: 14px;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  align-items: flex-end;
```

Comment:
You have already added property "align-items: flex-end" in parent block (line 45). Is it neccessary to repeat here?